test/test_process_pdos.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
```

Comment:
This file shouldn't be part of this PR? I think you need to pull from upstream master and rebase.